lib/src/presentation/views/movie_list.dart

```diff
@@ -26,9 +26,11 @@ class _MovieListState extends State<MovieList> {
   Future<List<Movie>> getMoviesList() async {
     List<Movie> movies = await movieRepository.fetchMovies();
 
-    return movies.map((movie) {
-      return movie;
-    }).toList();
+    return movies.map(
+      (movie) {
+        return movie;
+      },
+    ).toList();
```

Comment:
Why not just `return await movieRepository.fetchMovies()`? Or even `return movieRepository.fetchMovies()` without the `async` above